identity/client/src/pages/users/detail/authorization/List.tsx

```diff
@@ -47,23 +112,150 @@ const List: FC<AuthorizationsListProps> = ({ user, loadingUser }) => {
 
   return (
     <>
-      <EntityList
-        title={t("Authorizations assigned to user")}
-        data={authorizations == null ? [] : authorizations.items}
-        headers={[{ header: t("ResourceType"), key: "resourceType" }]}
-        onEntityClick={showAuthorizationDetails}
-        addEntityLabel="Assign authorization"
-        onAddEntity={() => {}}
-        menuItems={[
-          {
-            label: t("Delete"),
-            onClick: () => {},
-            isDangerous: true,
-            icon: TrashCan,
-          },
-        ]}
-        loading={loadingAuthorizations}
-      />
+      <TableContainer
+        title="Authorizations"
+        description="Authorizations's of the user"
+      >
+        <DataTable<Authorization> rows={rows} headers={headers}>
+          {({
+            rows,
+            headers,
+            getHeaderProps,
+            getRowProps,
+            getTableProps,
+          }: DataTableRenderProps<Authorization, unknown[]>) => (
+            <div>
+              {/* Table Toolbar with Add Row Button */}
+              <TableToolbar>
+                <TableToolbarContent
+                  style={{
+                    display: "flex",
+                    justifyContent: "flex-end",
+                    gap: "16px",
+                  }}
+                >
+                  <TableToolbarSearch />
+                  <button
+                    onClick={() =>
+                      handleOpenPatchModal({
+                        ownerKey: user.key,
+                        resourceType: null,
+                        permissionType: null,
+                        currentAuthorizations: rows,
+                      })
+                    }
+                    style={{
+                      border: "none",
+                      background: "#0f62fe",
+                      color: "white",
+                      cursor: "pointer",
+                      padding: "8px 16px",
+                      borderRadius: "4px",
+                      display: "flex",
+                      alignItems: "center",
+                    }}
+                  >
+                    <Add style={{ marginRight: "8px" }} />
+                    Add Authorizations
+                  </button>
```

Comment:
Please don't use a custom button, that's why we use the Carbon design system. [We already have a button built-in](https://react.carbondesignsystem.com/?path=/docs/components-button--overview#button-rendericon)